src/devana/syntax_abstraction/organizers/sourcefile.py

```diff
@@ -197,6 +211,23 @@ def from_cursor(cls, cursor: cindex.Cursor, parent: Optional = None,
             return None
         return cls(cursor, parent, configuration)
 
+    @classmethod
+    @init_params(skip={"cls", "parent", "namespace"})
+    def from_params( # pylint: disable=unused-argument
+            cls,
+            parent: Optional = None,
+            content: Optional = None,
+            namespace: Optional = None,
```

Comment:
Why namespace is ignored?

Reply:
Because the namespace in `SourceFile` is only overwritten by the getter, which always returns `None` (it's SourceFile after all). We can get rid of this parameter, but then the order of the parameters from inheritance will change and pylint will scream 🖐️🤬🖕🪓, so we will either ignore this pylint warning or the parameter itself.

Reply:
f namespace doesn't have a setter then it shouldn't be possible to set. We shouldn't lie to the API user.